src/mesh/mesh.cpp

```diff
@@ -917,6 +917,161 @@ void Mesh::ApplyUserWorkBeforeOutput(Mesh *mesh, ParameterInput *pin,
   }
 }
 
+void Mesh::BuildCommunicationBuffers() {
```

Comment:
Maybe add a comment what differentiate `BuildCommunicationBuffers` from` BuildBoundaryBuffers`?

Reply:
I guess we could just call this `BuildBoundaryBuffers` since it is just building the tag map then calling `BuildBoundaryBuffers`.

Reply:
done.